### ISSUE TITLE
nix-env error at line 57

### ISSUE THREAD

**Issue by rhacker** [NixOS/nix#502]
```sh
nix-env -f channels/nixpkgs/pkgs -qa -A oraclejdk
nix-env: src/libexpr/eval.hh:57: void nix::Bindings::push_back(const nix::Attr&): Assertion `size_ < capacity' failed.
zsh: abort      nix-env -f channels/nixpkgs/pkgs -qa -A oraclejdk
```

I guess this is memory usage related problem .

**Comment by lethalman** [comment]
On 11/03/2015 10:36, Dung Quang wrote:
> nix-env -f channels/nixpkgs/pkgs -qa -A oraclejdk
channels/nixpkgs, not channels/nixpkgs/pkgs . But that error is interesting.

**Comment by edolstra** [comment]
I think this was fixed in c2a8b5c42d37e129a893ee42479dbf395fdf7d4c.

**Comment by rhacker** [comment]
it's fixed for me. Thanks :)
